.github/workflows/codeql-analysis.yml

```diff
@@ -18,7 +18,7 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [master,2_1_X]
+    branches: [master,2_1_X,pr-*]
```

Comment:
I don't understand such changes to GH workflow: has anything changed lately? Otherwise, such changes are not required to let the workflows run on PRs.

Reply:
When you are workin on a fork of the codebase, and push changes to a branch other than master, CI jobs will not run on your fork for that branch unless this change is in place. PR rules are separate from this and are not affected.

Reply:
Ok, I see, thanks.